### ISSUE TITLE
Release notes

### ISSUE THREAD

**Issue by pjaaskel** [CHIP-SPV/chip-spv#165]
I started a release notes, please add whatever info you feel should be there https://github.com/CHIP-SPV/chip-spv/blob/Release-0.9/docs/release_notes/release-0.9.txt

This is on purpose in a format that's easy to copy-paste to a spam email. I usually do this for OpenASIP and PoCL and send it around.

@pvelesko @Kerilk @franz @linehill

**Comment by pjaaskel** [comment]
Please all check that you are happy with the dense release note/spam text.

**Comment by franz** [comment]
> ... oneAPI Level Zero as the low-level runtime options.

I'd rephrase it to "low-level runtime backends".

> ... project, thus bumps the road

... project, thus expect bumps in the road

> multiple GPU devices [list].

should this be a URL, or do we just list it in the text ? We can add Intel Skylake HD Graphics 510 (Henry's machine) and whatever Paulius uses for tests (IIRC some Iris Pro, not sure which one).
